lib/linter-eslint.js

```diff
@@ -138,6 +143,11 @@ export default {
       try {
         results = JSON.parse(output).results
         messages = results[0].messages || []
+        // clobber ignored message based on settings
+        if (messages.length === 1 && !!~messages[0].message.indexOf('ignored because of your .eslintignore file') &&
```

Comment:
I like this a lot: https://github.com/MoOx/eslint-loader/blob/master/index.js#L30 what do you think?